EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1230,273 +1230,273 @@ QUEST_LV_0300_20160310_001229	Even if you can't trust me you should trust Vaidot
-QUEST_LV_0300_20160310_001259	But, I don't know how I should destroy it..{nl}I can't think of anything at the moment.{nl}
-QUEST_LV_0300_20160310_001260	But, wait..{nl}I can feel the spirit of the demons in you.{nl}
-QUEST_LV_0300_20160310_001261	The marble which you have..{nl}Is the demon's spirit locked inside it?{nl}
-QUEST_LV_0300_20160310_001262	So that was it.{nl}Maybe we can use this to destroy the vicious idol.{nl}
-QUEST_LV_0300_20160310_001263	I am telling you to purify the spirit of the demons and use the power.{nl}But, we don't have enough amount of the spirits to detroy the idol.{nl}
-QUEST_LV_0300_20160310_001264	Can you fill the demon's spirit little more?{nl}As you did so far, when you defeat the demons, the spirits would be aborbed automatically.
-QUEST_LV_0300_20160310_001265	The demons are strong so.. I guess the sprits are also strong.{nl}With this much power, my sacred energy would be lacking.
-QUEST_LV_0300_20160310_001266	We should somehow borrow the power of the divine water..{nl}But, as I told you before, I have nothing at the moment.{nl}
-QUEST_LV_0300_20160310_001267	I am so sorry, but can you get me the divine water from the Priest Master at Klaipeda?{nl}If we have enough divine water, we would be able to purify the collapsed spirits of the demons. 
-QUEST_LV_0300_20160310_001268	Please use the statue of the goddess of Bakarine near Baime Fountains. {nl}You can quickly come back from Klaipeda.
-QUEST_LV_0300_20160310_001269	With this much, you would be able to purify the soul marble.{nl}Please wait a bit.
-QUEST_LV_0300_20160310_001270	As the power of the spirit is strong, the purified power is also strong.{nl}This power would be able to destroy the vicious idol.{nl}
-QUEST_LV_0300_20160310_001271	Put the power of the soul marble into the purified evil idol.{nl}I am sure the idol would not withstand..
-QUEST_LV_0300_20160310_001272	Finally!{nl}Now, we would be free from the black wall.{nl}
-QUEST_LV_0300_20160310_001273	It's been such a long time..{nl}I think the prayer to the goddess was not meaningless..
-QUEST_LV_0300_20160310_001274	Now, the main culprit is the priest of the religious body of the truth, Dominicas..{nl}Everything will be over when we drive him out.{nl}
-QUEST_LV_0300_20160310_001275	Please lend me your power little more.{nl}Even if we have ways to get him, the villagers are all seduced and due to long torture, we can't use our power..
-QUEST_LV_0300_20160310_001276	He seems to have some power to control the plants freely.{nl}Be careful since he is not to be underestimated.
-QUEST_LV_0300_20160310_001277	Dominicas.. Ran away��{nl}
-QUEST_LV_0300_20160310_001278	But, it's okay.{nl}It's a big harvest that we drove out the religious body of the truth from Elgos Monastery.{nl}
+QUEST_LV_0300_20160310_001233	I am talking about the scripture from the Tree of Truth sect left by Dominikas.{nl}It turns out that impious stuff is still remaining in the monastery.{nl}
```

Comment:
"I'm talking about the scripture that Dominikas left from the Tree of Truth sect. Impious remnants like that still remain in the monastery."